pkg/sdk/testint/external_volumes_gen_integration_test.go

```diff
@@ -507,41 +394,37 @@ func TestInt_ExternalVolumes(t *testing.T) {
 		props, err := client.ExternalVolumes.Describe(ctx, id)
 		require.NoError(t, err)
 
-		trimmedProperties := trimProperties(t, props)
-		assert.Equal(t, 5, len(trimmedProperties))
-		assert.Contains(t, trimmedProperties, ExternalVolumePropNameValue{Name: "COMMENT", Value: comment})
-		assert.Contains(t, trimmedProperties, ExternalVolumePropNameValue{Name: "ALLOW_WRITES", Value: strconv.FormatBool(allowWrites)})
-		assert.Contains(t, trimmedProperties, ExternalVolumePropNameValue{Name: "ACTIVE", Value: ""})
-		assert.Contains(
-			t,
-			trimmedProperties,
-			ExternalVolumePropNameValue{
-				Name: "STORAGE_LOCATION_1",
-				Value: fmt.Sprintf(
-					`{"NAME":"%s","STORAGE_PROVIDER":"GCS","STORAGE_BASE_URL":"%s","ENCRYPTION_TYPE":"%s"}`,
-					gcsStorageLocationsNoneEncryption[0].GCSStorageLocationParams.Name,
-					gcsStorageLocationsNoneEncryption[0].GCSStorageLocationParams.StorageBaseUrl,
-					gcsStorageLocationsNoneEncryption[0].GCSStorageLocationParams.Encryption.Type,
-				),
-			},
-		)
-		assert.Contains(
-			t,
-			trimmedProperties,
-			ExternalVolumePropNameValue{
-				Name: "STORAGE_LOCATION_2",
-				Value: fmt.Sprintf(
-					`{"NAME":"%s","STORAGE_PROVIDER":"%s","STORAGE_BASE_URL":"%s","STORAGE_AWS_ROLE_ARN":"%s","STORAGE_AWS_EXTERNAL_ID":"%s","ENCRYPTION_TYPE":"%s","ENCRYPTION_KMS_KEY_ID":"%s"}`,
-					s3StorageLocations[0].S3StorageLocationParams.Name,
-					s3StorageLocations[0].S3StorageLocationParams.StorageProvider,
-					s3StorageLocations[0].S3StorageLocationParams.StorageBaseUrl,
-					s3StorageLocations[0].S3StorageLocationParams.StorageAwsRoleArn,
-					*s3StorageLocations[0].S3StorageLocationParams.StorageAwsExternalId,
-					s3StorageLocations[0].S3StorageLocationParams.Encryption.Type,
-					*s3StorageLocations[0].S3StorageLocationParams.Encryption.KmsKeyId,
-				),
+		parsedExternalVolumeDescribed, err := helpers.ParseExternalVolumeDescribed(props)
+		require.NoError(t, err)
+		expectedParsedExternalVolumeDescribed := helpers.ParsedExternalVolumeDescribed{
```

Comment:
We now have such assertions in the `objectassert` package, but I think we can move this on our side.